fastlane_core/spec/fastlane_folder_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe FastlaneCore::FastlaneFolder do
+  let (:fastfile_name) { "Fastfile" }
+  describe "#path" do
+    it "returns the fastlane path if it exists" do
+      expect(FastlaneCore::FastlaneFolder.path).to eq("./fastlane/")
+    end
+
+    it "returns nil if the fastlane path isn't available" do
+      Dir.chdir("/") do
+        expect(FastlaneCore::FastlaneFolder.path).to eq(nil)
+      end
+    end
+
+    it "returns the path if fastlane is being executed from the fastlane directory" do
+      Dir.chdir("fastlane") do
+        expect(FastlaneCore::FastlaneFolder.path).to eq('./')
+      end
+    end
+  end
+
+  describe "#fastfile_path" do
```

Comment:
Interesting, I thought it would be `#`. We don't use the `.` across the code base currently. Is that something you'd want to change? I don't feel like it's worth the effort.